solr/contrib/README.md

```diff
@@ -0,0 +1,31 @@
+Apache Solr HDFS Contrib module
```

Comment:
Should this be under the hfs contrib directory instead of top level?

Reply:
Agree w/ @madrob - there is already a `solr/contrib/hdfs/README.md` as well? Should be consolidated.

Reply:
Fixed (https://github.com/apache/solr/pull/324/commits/e0a4b5bcd355554e2a59fc240123338e05a97716)